src/onboarding/registration/EnableBiometry.tsx

```diff
@@ -122,9 +122,9 @@ export default function EnableBiometry({ navigation }: Props) {
 
   return (
     <ScrollView style={styles.contentContainer}>
-      <SafeAreaView style={styles.container}>
+      <SafeAreaView style={showGuidedOnboarding ? styles.containerLeftAligned : styles.container}>
```

Comment:
nit: I don't think this style change needs to be part of the experiment. It can just be universal. check with @nityarao though to be sure!